README.md

```diff
@@ -2,6 +2,18 @@
 
 [![Build Status](https://gpuci.gpuopenanalytics.com/job/rapidsai/job/gpuci/job/rmm/job/branches/job/rmm-branch-pipeline/badge/icon)](https://gpuci.gpuopenanalytics.com/job/rapidsai/job/gpuci/job/rmm/job/branches/job/rmm-branch-pipeline/)
 
+**NOTE:** For the latest stable [README.md](https://github.com/rapidsai/rmm/blob/main/README.md) ensure you are on the `main` branch.
+
+## Resources
+
+- [RMM Reference Documentation](https://docs.rapids.ai/api/rmm/stable/): Python API reference, tutorials, and topic guides.
+- [librmm Reference Documentation](https://docs.rapids.ai/api/librmm/stable/): C/C++ CUDA library API reference.
+- [Getting Started](https://rapids.ai/start.html): Instructions for installing RMM.
+- [RAPIDS Community](https://rapids.ai/community.html): Get help, contribute, and collaborate.
+- [GitHub repository](https://github.com/rapidsai/rmm): Download the cuDF source code.
+- [Issue tracker](https://github.com/rapidsai/rmm/issues): Report issues or request features.
```

Comment:
```suggestion
- [Issue tracker](https://github.com/rapidsai/rmm/issues): Report issues or request features.
- [Stream-ordered Allocation](https://developer.nvidia.com/blog/using-cuda-stream-ordered-memory-allocator-part-1/): More information about the semantics of stream-ordered allocation. 
```

Reply:
@jrhemstad This blog post isn't _about_ RMM as a library, it's a topic of background info that developers should know. It can definitely be linked in the README, but it doesn't feel appropriate for the Resources top-level section. I'd link it in the section titled "Stream-ordered Memory Allocation" below. If you're okay with that, let me know and I'll make the change.

Reply:
```suggestion
- [Issue tracker](https://github.com/rapidsai/rmm/issues): Report issues or request features.
- [Fast, Flexible Allocation for NVIDIA CUDA with RAPIDS Memory Manager](https://developer.nvidia.com/blog/fast-flexible-allocation-for-cuda-with-rapids-memory-manager/): NVIDIA Developer Blog post about RMM
```

Reply:
I tackled this in 4d675f2b4ec160de16cdc74ee7b842a47db79d80. Feel free to suggest changes / approve if you're happy with it.

Reply:
If that's what you have in mind for "Resources" then sure. 

I had thought this section was more of a "Further Reading" as just a collection of links to things anyone interested in RMM would also be interested in.

Reply:
I try to keep the "Resources" section brief since it's a top level thing -- just a set of links to crucial project information, like how to acquire, use, report issues with, and contribute to the code. "Further Reading" definitely fits in the README, just maybe not in the core project info.